src/js/pages/UniversePage.js

```diff
@@ -52,13 +51,7 @@ class UniversePage extends mixin(TabsMixin) {
   }
 
   render() {
-    return (
-      <Page
```

Comment:
Having deleted this we can also delete all the methods from the class and inherit it directly from `React.Component` I tried and it still worked :)